### PR TITLE
Adds resume section to the website & replaces project section. 

### PR DESCRIPTION
**Pull request by hailed748**
Modified the website so that there is now an online version of my resume. Also turns blog section into the projects section. Went through and added more consistent spacing on my style.css file.

### CODE REVIEW

**Comment by LukeGMTravis** [comment]
Starting review

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -31,140 +35,176 @@ <h1>About Me</h1>
...
+        <section>
+        <h1>Skills & Experience</h1>
+        <hr class="short">
+        <h2>Education</h2>
+        <div class"resume-container">
+            <div class="section">
+                <div class="header">
+                    <h3>Massachusetts Institue of Techonology</h3>
+                    <h4>Candidate for BS in EECS</h3>
+                    <p>Cambridge,MA<p>
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:56]: This isn't a code nit but just a question: I'm noticing that in some places you have a space after a comma in the text and in some places you don't, is this intentional?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -31,140 +35,176 @@ <h1>About Me</h1>
...
+                    <p>Sep 2018 - Jun 2020</p>
+                </div>
+                <div class="info">
+                    <p class="courses">
+                        <span class="first">Coursework:</span> Mathematics for CS, Fundamentals of Python, Intro to Computational Thinking, Data Structures & Algorithims,
+                        Intro Embeded Systems & IoT, Differential Equations, Clasical Mechanics, Electricity & Magnetisim
+                    </p>
+                    <p class="courses">
+                        <span class="first">Knowledgeable:</span> Python,HTML,CSS,React Native
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:65]: Same question about commas and spaces

Comment (LukeGMTravis) [reply portfolio/src/main/webapp/index.html:65]: And same in a couple places below too

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -31,140 +35,176 @@ <h1>About Me</h1>
...
+              <span><h4 class="card-topic">School</h4></span>
               <span class="date">
                 <h3 class="day">17</h3>
                 <h4 class="month">Jun</h4>
               </span>
             </span>
             <span class="card-summary">
-              <h1 class="card-title">First Week at Google </h1>
-              <h2 class="card-subtitle">Did I fail miserably?</h2>
+              <h1 class="card-title">CaringCart </h1>
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:165]: Extra space

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (hailed748) [reply portfolio/src/main/webapp/index.html:56]: That is just me being inconsistent, I will fix them.

**Pull request merged**